### PR TITLE
documentation/ZENKO-3057_XDM_UI_delete_a_bucket

### PR DESCRIPTION
**Pull request by wabernat**
Added "delete a bucket" file and supporting graphics and index files.

### CODE REVIEW

**Review by miniscruff — APPROVED** [review]

File: solution-base/build.sh [hunk 1]
```diff
@@ -32,9 +32,7 @@ SOLUTION_ENV='SOLUTION_ENV'
 export KUBEDB_OPERATOR_TAG=$(grep kubedb/operator: deps.txt | awk -F ':' '{print $2}')
 export KUBEDB_NAMESPACE=${SOLUTION_ENV}
 export KUBEDB_SERVICE_ACCOUNT=kubedb-operator
-export KUBEDB_IMAGE_NAME=operator
```
Comment (jonathan-gramain) [inline solution-base/build.sh]: Was it intended to commit those changes? It does not seem related to documentation, hence asking.

Comment (miniscruff) [reply solution-base/build.sh]: looks like one of my commits from solution work, not sure how it ended up here

Comment (wabernat) [reply solution-base/build.sh]: I am confident that this was the result of some Git buffoonery on my part. I'll see if I can roll it back.

Comment (wabernat) [reply solution-base/build.sh]: I just copy/pasted this back to "before", figuring it would be easier and more predictable than my trying to track down and modify the bad commit via Git magic. If this is a problem, let me know, and I'll work on a cleverer response.

Comment (wabernat) [reply solution-base/build.sh]: Hoo boy. Scrolling down, I see this is worse than I thought. I'll rebase and hope Ronnie's mods are taken care of.

Comment (wabernat) [reply solution-base/build.sh]: I think that took care of it.

**Comment by miniscruff** [comment]
@wabernatScality do you think you could squash the commits into one? I can give you the git commands if you need help.

**Comment by wabernat** [comment]
Squashed.

**Comment by tmacro** [comment]
@wabernatScality will you squash the last commit? otherwise LGTM

**Review by tcarmet — APPROVED** [review]
